import/popups/AuthDialog.qml

```diff
@@ -12,92 +12,121 @@
-            EnterKey.onClicked: root.accept()
-        }
-
-        TextSwitch {
-            id: remember
-
-            // If credentials are already remember removing them via this is not feasibible
-            // Better to hide the whole checkbox.
-            visible: !privateBrowsing && !(root.remember && root.remember.checked)
-            checked: false
-
-            //: Remember entered credentials for later use
-            //% "Remember credentials"
-            text: qsTrId("sailfish_components_webview_popups-remember_credentials")
-        }
-
-        Label {
-            x: Theme.paddingLarge
-            width: parent.width - Theme.paddingLarge * 2
-            visible: privateBrowsing
-            wrapMode: Text.Wrap
-            color: Theme.highlightColor
-
-            //: Description label for user when private mode active (entered credentials are not saved)
-            //% "Credential storage is not available in private browsing mode"
-            text: qsTrId("sailfish_components_webview_popups-la-private_browsing_credentials_description")
+Dialog {
+    id: dialog
+
+    property alias hostname: auth.hostname
+    property alias realm: auth.realm
+    property alias passwordOnly: auth.passwordOnly
+    property alias privateBrowsing: auth.privateBrowsing
+
+    property alias usernameVisible: auth.usernameVisible
+    property alias usernameAutofocus: auth.usernameAutofocus
+    property alias usernamePrefillValue: auth.usernamePrefillValue
+    property alias passwordPrefillValue: auth.passwordPrefillValue
+    property alias rememberVisible: auth.rememberVisible
+    property alias rememberPrefillValue: auth.rememberPrefillValue
+
+    property alias usernameValue: auth.usernameValue
+    property alias passwordValue: auth.passwordValue
+    property alias rememberValue: auth.rememberValue
+
+    property alias acceptText: auth.acceptText
+    property alias cancelText: auth.cancelText
+    property alias title: auth.title
+    property alias preventDialogsVisible: auth.preventDialogsVisible
+    property alias preventDialogsPrefillValue: auth.preventDialogsPrefillValue
+    property alias preventDialogsValue: auth.preventDialogsValue
+
+    AuthPopupInterface {
+        id: auth
+
+        anchors.fill: parent
+
+        usernameValue: username.text
+        passwordValue: password.text
+        rememberValue: remember.checked
+
+        //: Text on the Accept dialog button that accepts browser's auth request
+        //% "Log In"
+        acceptText: qsTrId("sailfish_components_webview_popups-he-accept_login")
+
+        onAccepted: dialog.accept()
+        onRejected: dialog.reject()
+
+        UserPromptUi {
+            anchors.fill: parent
+            dialog: dialog
+            popupInterface: auth
+
+            Column {
+                width: parent.width
+
+                Label {
+                    x: Theme.paddingLarge
```

Comment:
For sake of being good citizen we should promote in examples ```Theme.horizontalPageMargin``` when content touching edges.

Reply:
done for the popups ones